docs/openapi.yaml

```diff
@@ -652,10 +693,126 @@ components:
           type: string
     ImportCandidate:
       type: object
-      description: Complete ImportCandidate entity
+      description: Complete Import candidate entity
       properties:
```

Comment:
allof: does not work?

Reply:
i discussed it with Kjetil, allOf make swagger object look weird